server/src/get-options.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Shouldn't we add `set -euo pipefail`?

Reply:
The completion doesn't work with `set -e`.
https://man.archlinux.org/man/set.1p#APPLICATION_USAGE